dadi/frontend/components/FieldImage/FieldImageRoutes.js

```diff
@@ -0,0 +1,89 @@
+'use strict'
+
+const config = require(paths.config)
+const AWS = require('aws-sdk')
+
+const ImageFieldRoute = function (app) {
+
+  if (config.get('FieldImage.s3.enabled')) {
+    this.initAWS()
+    this.S3 = this.initS3()
+
+    app.post({
+      name: 'images3sign', // This allows us to reuse the auth request
+      path: '/fields/image/s3/sign'
+    }, 
+    (req, res, next) => {
+      res.header('Content-Type', 'application/json')
+      let response = {route: req.route.name}
+
+      if (req.isAuthenticated()) {
+        if (req.params && req.params.fileName) {
+
```

Comment:
minor: we've not been using blank lines at the beginning of function declarations or any other blocks.